File: Easy/MissingNumber.py
class Solution:
    def missingNumber(self, nums):
        '''
        gauss = len(nums) * (len(nums) + 1) / 2
        ungauss = sum(nums)
        return int(gauss) - ungauss
        '''
        
        return int(len(nums) * (len(nums) + 1) / 2) - sum(nums)

File: Java/Easy/MissingNumber.java
class Solution {
    public int missingNumber(int[] nums) {
        int bigSum = 0;
        int smallSum = 0;
        int i = 0;
        for(i = 0; i < nums.length; i++){
            bigSum += i;
            smallSum += nums[i];
        }
        return bigSum + i - smallSum;
    }
}